components/d2l-user.js

```diff
@@ -0,0 +1,196 @@
+import '@brightspace-ui/core/components/colors/colors';
```

Comment:
```suggestion
import '@brightspace-ui/core/components/colors/colors.js';
```
I can never remember when this causes issues or not

Reply:
Ah yup, my bad. Missed this in the big copy paste